redwoods-insurance-salesforce/force-app/main/default/lwc/incidentAudioPlayer/__tests__/data/mockAudioFile.json

```diff
@@ -0,0 +1 @@
+[{ "Id": "file1" }]
```

Comment:
Shouldn't the value be a `ContentVersion` Id?

Reply:
No. they're mocks. It doesn't matter what the values are.

Reply:
As in the other PR, this should be real data.

Reply:
Again, I disagree. We're doing our developers a disservice if they cannot clearly see what is mock data and what is not. I understand your need for consistency across the apps, but in this case it's a developer relations misstep.

Reply:
This is about how to present proper data shapes for proper testing. That's why we do it consistently - also outside developer relations - for that.

Reply:
Proper testing uses mocks. Which means the data inside the mock is irrelevant. Therefore, proper testing using mocks, utilizes obviously fake data to illustrate that the data is a mock. 

Also I have no idea what you mean by 'outside developer relations'